src/scs/slsa-framework.md

```diff
@@ -0,0 +1,22 @@
+# SLSA Framework
+
+Supply chain Levels for Software Artifacts (SLSA) is a security framework for tampering prevention, integrity improvement and securing packages and infrastructure of a project.
+
+## Levels of assurance
+
+One of the requirements for the solution is to reach SLSA Level 4 and even go beyond that. This requires a lot of process changes as well as technical work. Current status for Thincompute project is visualized at the separate wiki page: Supply Chain Security 
```

Comment:
Currently, the **Ghaf** project is visualized on...
@brianmcgillion am I right that here we do not want to use Thincompute as a project? I replaced it with Ghaf.

Reply:
correct